content/2. Ondersteunende-informatie/3. Processen/6. Beheerproces/3. Gebruiken beheersysteem/Git Flow.md

```diff
@@ -0,0 +1,88 @@
+---
+title:
+  - Git Flow
+taxonomie:
+  - bg-24.1.Branching-strategiën.OI
+  - bg-24.2.Git-Flow.OI
+---
+
+## Wat is Git Flow?
+Git Flow is een van de populaire [[Branching strategiën]] voor Git, geïntroduceerd door Vincent Driessen in 2010. Het helpt bij het beheren van releases en het organiseren van de ontwikkeling van softwareprojecten. Git Flow gebruikt de volgende branches.
+![[git-flow.png]]
```

Comment:
Deze image zou falen in de pipeline doordat het niet begint met een 4C/ID component, dus voor dit zou dat OI_git_flow_1.png kunnen zijn (Ik zag al dat er twee images waren voor gitflow, dus dan kan die bijvoorbeeld OI_git_flow_2.png kunnen heten).